govtool/frontend/src/context/wallet.tsx

```diff
@@ -829,8 +897,13 @@ const CardanoProvider = (props: Props) => {
         const myWithdrawal = BigNum.from_str(amount);
         const withdrawals = TreasuryWithdrawals.new();
         withdrawals.insert(treasuryTarget, myWithdrawal);
-        // Create new treasury withdrawal gov act
-        const treasuryAction = TreasuryWithdrawalsAction.new(withdrawals);
+        const guardrailScript = PlutusScript.from_bytes_v3(
+          Buffer.from(GUARDRAIL_SCRIPT, "hex"),
+        );
+        const treasuryAction = TreasuryWithdrawalsAction.new_with_policy_hash(
```

Comment:
Technically it is possible to NOT have a guardrails script on-chain
although, it is probably unlikely that we wont have one

Reply:
I would keep this change and discuss the alternative solutions post hard fork, wdyt?

Reply:
I'll do that. We can update that anytime :saluting_face: